### PR TITLE
Add .cls files

### PR DESCRIPTION
**Pull request by isc-ddo**

### CODE REVIEW

File: Setup/src/cls/GeoResponse.cls [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/// This class holds GeoJSON information retrieved from an API
+Class Demo.GeoResponse Extends (%Persistent, Ens.Util.MessageBodyMethods)
```
Comment (zkrowiak) [inline Setup/src/cls/GeoResponse.cls]: these are production messages, why don't they extend request and response?

File: Setup/src/cls/GeoRequest.cls [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/// This class holds information needed to request GeoJSON data from an API
+Class Demo.GeoRequest Extends (%Persistent, Ens.Util.MessageBodyMethods)
```
Comment (zkrowiak) [inline Setup/src/cls/GeoRequest.cls]: these are production messages, why don't they extend request and response?

File: Setup/src/cls/PopulationRequest.cls [hunk 3]
```diff
@@ -0,0 +1,13 @@
+/// This class holds information needed to request population density information from an API
+Class Demo.PopulationRequest Extends (%Persistent, Ens.Util.MessageBodyMethods)
```
Comment (zkrowiak) [inline Setup/src/cls/PopulationRequest.cls]:
```suggestion
Class Demo.PopulationRequest Extends (Ens.Request)
```

File: Setup/src/cls/PopulationResponse.cls [hunk 4]
```diff
@@ -0,0 +1,7 @@
+/// This class holds population density information retrieved from an API
+Class Demo.PopulationResponse Extends (%Persistent, Ens.Util.MessageBodyMethods)
```
Comment (zkrowiak) [inline Setup/src/cls/PopulationResponse.cls]:
```suggestion
Class Demo.PopulationResponse Extends (Ens.Request)
```

File: Setup/src/cls/ToCensusDS.cls [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+Parameter INVOCATION = "Queue";
+
+/// Calls an external API to get the population density for a particular GeoJSON location.
+Method GetPopulationDensity(pRequest As Demo.PopulationRequest, Output pResponse As Demo.PopulationResponse) As %Status
+{
+  try {
+      // Prepare and log the call
+      // Append the city to the URL configured for adapter
+      set tURL= ..Adapter.URL _ "?get=NAME,B00001_001E&for=block%20group:" _ pRequest.BlockGroup _ "&in=state:" _ pRequest.State _
```
Comment (zkrowiak) [inline Setup/src/cls/ToCensusDS.cls]:
```suggestion
      set tURL= ..Adapter.URL _          // an underscore concatenates strings
       "?get=NAME,B00001_001E&for=block%20group:" _
       pRequest.BlockGroup _
       "&in=state:" _
       pRequest.State _
       
```

File: Setup/src/cls/PopulationResponse.cls (hunk 4 above)
Comment (zkrowiak) [reply Setup/src/cls/PopulationResponse.cls]: also, is the Ens.Util.MessageBodyMethods class necessary? (this goes for all of these classes.

File: Setup/src/cls/GeoResponse.cls (hunk 1 above)
Comment (zkrowiak) [inline Setup/src/cls/GeoResponse.cls]:
```suggestion
Class Demo.GeoResponse Extends (Ens.Response)
```

File: Setup/src/cls/ToCensusDS.cls (hunk 5 above)
Comment (zkrowiak) [reply Setup/src/cls/ToCensusDS.cls]: this makes it prettier -- but it would need to be tested.

File: Setup/src/cls/ToCensusDS.cls [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+      "&in=county:" _ pRequest.County _ "&in=tract:" _ pRequest.Tract _ "&key=6c59dddebe1002a5d5a9848e404e19e5f1614ce9"
```
Comment (zkrowiak) [inline Setup/src/cls/ToCensusDS.cls]:
```suggestion
      "&in=county:" _
      pRequest.County _
      "&in=tract:" _
      pRequest.Tract _ 
      "&key=6c59dddebe1002a5d5a9848e404e19e5f1614ce9"
```

File: Setup/src/cls/ToCensusGeoServiceBO.cls [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+Parameter INVOCATION = "Queue";
+
+/// Retrieves GeoJSON data from the USCensus using longitude and latitude.
+Method GetGeoInfo(pRequest As Demo.GeoRequest, Output pResponse As Demo.GeoResponse) As %Status
+{
+  try {
+      // Prepare and log the call
+      // Append the city to the URL configured for adapter
+      set tURL= ..Adapter.URL _ "query?geometry=" _ pRequest.Longitude _ "," _ pRequest.Latitude _ "&geometryType=esriGeometryPoint&outfields=*&returnGeometry=true&returnIdsOnly=false&inSR=4326&f=geojson" //..Adapter.URL_"?q="_pRequest.City
```
Comment (zkrowiak) [inline Setup/src/cls/ToCensusGeoServiceBO.cls]:
```suggestion
      set tURL= ..Adapter.URL _ 
      "query?geometry=" _ 
      pRequest.Longitude _ 
      "," _ 
      pRequest.Latitude _ "&geometryType=esriGeometryPoint&outfields=*&returnGeometry=true&returnIdsOnly=false&inSR=4326&f=geojson" //..Adapter.URL_"?q="_pRequest.City
```

**Review by isc-zkrowiak — CHANGES_REQUESTED** [review]

**Comment by zkrowiak** [comment]
also should the readme point to the gettin started site?

**Comment by DuongDo-Intersystems** [comment]
@zkrowiak README will be point to Getting Started site after these changes get merged. I don't want to mess us the Github.

**Comment by DuongDo-Intersystems** [comment]
@zkrowiak I test your new change and it works. I will question Jenny later why she does not use Ens.Request and Ens.Response when we actually use this sample for Getting Started page. Please review and merge if you feel good about it.

**Review by isc-zkrowiak — APPROVED** [review]

**Pull request merged**